db/migrate/20160226014344_create_pallets.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePallets < ActiveRecord::Migration
+  def change
+    create_table :pallets do |t|
+      t.references :cycle_count, index: true, foreign_key: true
+      t.string :pallet_number, null: false
+      t.boolean :pallet_in_grossman, default: false
+      t.boolean :pallet_in_location, default: false
```

Comment:
I was thinking ahead based on querying against Grossman.  If it can find the pallet in that location in Grossman then it'll return true to confirm it was found in that location.